Resources/Prototypes/_LostParadise/Flavors/flavors.yml

```
@@ -26,4 +26,14 @@
-  description: flavor-complex-flower-tea
+  description: flavor-complex-flower-tea
+
+- type: flavor
+  id: lppcargogargleblaster
+  flavorType: Complex
+  description: flavor-complex-lpp-cargo-gargle-blaster
+
+- type: flavor
+  id: lppmiruvor
+  flavorType: Complex
+  description: flavor-complex-lpp-miruvor
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)